tests/test_help_command.py

```diff
@@ -3,12 +3,13 @@
 from onebuild.main import run
 
 USAGE_HELP_MESSAGE = \
-    "usage: 1build [-h] [-l] [command]\n\n" \
+    "usage: 1build [-h] [-l] [-v] [command]\n\n" \
```

Comment:
Maybe this is a good place to try python's 
`'''some_mulltiline_text'''` style?

Please give a try – if possible.

Reply:
Yes, updated the required the string as `""" multi-line String"""`

Reply:
👍